pages/pipelines/conditionals.md.erb

```diff
@@ -1,9 +1,21 @@
 # Using Conditionals
 
-The `if` parameter provides the ability to run steps only when specific conditions are met. Using C-like expressions, boolean conditions are defined to restrict the running of steps to specific circumstances. Note that we do not support Build Meta-data for conditionals.
+Using conditionals, you can run builds or steps only when specific conditions are met. Define boolean conditions using C-like expressions. Note that you cannot use [Build Meta-data](/docs/pipelines/build-meta-data) in conditional expressions.
+
+You can define conditionals at the step level in your `pipeline.yml` or at the pipeline level in your Buildkite version control provider settings.
 
 {:toc}
 
+## Conditionals in Pipelines
+
+Use conditionals in your pipeline settings (_Repository Providers_) to run builds run builds only when expressions evaluates to `true`. For example, to run only when a branch is tagged `Approved`:
```

Comment:
What's the "Repository Providers" here about?

Reply:
```suggestion
Use conditionals in your pipeline settings (_Repository Providers_) to run builds only when expressions evaluates to `true`. For example, to run only when a branch is tagged `Approved`:
```

Reply:
Thanks for the :eyes: @sj26 - yup I mixed the Repository settings, will fix.